docs/detections/rules-ui-monitor.asciidoc

```diff
@@ -2,7 +2,19 @@
 [role="xpack"]
 == Monitor and troubleshoot rule executions
 
-To view a summary of all rule executions, such as failures and last execution
+The Rules page offers several ways to gain insight into how your detection rules are running:
```

Comment:
Another option here could be: 

`The Rules page offers several ways to gain insight into the status of your detection rules.`

Reply:
```suggestion
The Rules page offers several ways to gain insight into the status of your detection rules:
```